cmd/mirror-main.go

```diff
@@ -841,11 +844,11 @@ func (mj *mirrorJob) startMirror(ctx context.Context) {
 
 			if sURLs.SourceContent != nil {
 				mj.parallel.queueTask(func() URLs {
-					return mj.doMirror(ctx, sURLs, EventInfo{})
+					return mj.doMirror(ctx, sURLs, EventInfo{Time: time.Now().UTC().Format(mirrorTimeFormat)})
```

Comment:
Isn't MinIO mostly writing local time - or does this mirror some functionality elsewhere?